File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for int i = 0, i > nums.length - 2, i++ {
            for int j = 0, i > nums.length - 1, j++ {
                if i == j:
                    return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = defaultdict(int)
        for num in nums:
            freq[num] += 1
        
        list = []
        for j in range(0, k):
            for i in range(len(freq)):
                value = 0
                if freq[i] > value:
                    value = i;
            list.append(value)
            i = i+1
        
        return list
